pyproject.toml

```diff
@@ -48,6 +48,14 @@ CAPTURE_DATE_PREV = "2023-07-01"
 internal_columns = '"Job_Number","Job_Type"'
 external_columns = '"Job_Number","Job_Type"'
 
+<<<<<<< HEAD
```

Comment:
You've got a little rebase detritus here - just delete this whole segment (lines 51-58, inclusive)